client.go

```diff
@@ -181,3 +181,21 @@ func (c Client[Resource]) Create(ctx context.Context, resource Resource, options
 	}
 	return createdResource.ID, nil
 }
+
+// DeleteById deletes an element using the resource _id.
+func (c Client[Resource]) DeleteById(ctx context.Context, id string, options Options) error {
+	req, err := c.client.NewRequestWithContext(ctx, http.MethodDelete, id, nil)
+	if err != nil {
+		return err
+	}
+
+	if err := options.setOptionsInRequest(req); err != nil {
+		return err
+	}
+
+	resource := new(Resource)
+	if _, err := c.client.Do(req, resource); err != nil {
```

Comment:
The response of the resource is the resource itself?

Reply:
The response body is expected to be empty. So, it makes no difference where we store it. Maybe a `nil` would be more appropriate?

Reply:
Yes, it should be a nil if we does not want to handle the response body

Reply:
Done